content/docs/reference/pkg/packet/bgpsession.md

```diff
@@ -10,6 +10,12 @@ meta_desc: "Explore the BgpSession resource of the Packet package, including exa
 <!-- WARNING: this file was generated by Pulumi Docs Generator. -->
 <!-- Do not edit by hand unless you're certain you know what you are doing! -->
 
+<div class="note note-info" role="alert">
+    <p><b>
+        Please note that the Packet provider has been deprecated. The replacement resource is Equinix Metal <a href="/docs/reference/pkg/equinix-metal/bgpsession/">BGP session</a>.
```

Comment:
The anchor text for the new link seems to be separated and lowercase; however the actual title of the Equinix Metal provider item seems to be concatenated and Pascal cased. Is it possible to do that for these links? 

I don't think it's a deal breaker, but it would be nice to be consistent with the title for the destination link.